tests/test_utils.py

```diff
@@ -12,6 +15,22 @@ def test_import_module(self):
         datetime = import_module("datetime.datetime")
         self.assertTrue(hasattr(datetime, 'now'))
 
+    def test_pos_tagger(self):
+        pos_tagger = NLTKPOSTagger()
+        tokens = pos_tagger.tokenize("what time is it")
+
+        self.assertEqual(tokens, ['what', 'time', 'is', 'it'])
+
+    def test_stop_words(self):
+        stopwords_manager = NLTKStopWordsManager()
+        words = stopwords_manager.words("english")
+        test_case = set(["too"]) - set(words)
+
+        self.assertEqual(test_case, set([]))
+
+    def test_word_net(self):
+        wordnet = NLTKWordnet()
+        synsets = wordnet.synsets('test')
 
```

Comment:
The case for `test_word_net` was probably intended to assert something i'm guessing?